src/examples/colorpickers/ColorpickerDialogFormInput.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { parseToRgb, toColorString } from 'polished';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { ColorpickerDialog, IColor } from '@zendeskgarden/react-colorpickers';
+import { Field, Label, Input, InputGroup } from '@zendeskgarden/react-forms';
+
+const validHex = /^#(?:(?:[0-9A-F]{6}(?:[0-9A-F]{2})?)|(?:[0-9A-F]{3})(?:[0-9A-F]?))$/iu;
+
+const StyledField = styled(Field)`
+  display: inline-block;
+  text-align: ${props => (props.theme.rtl ? 'right' : 'left')};
+`;
+
+const StyledInput = styled(Input)`
+  max-width: ${props => props.theme.space.base * 26}px;
```

Comment:
Let's apply a base-4 max-width that accommodates the widest possible value, which I believe would be `#AAAAAAAA`. I think `base * 27` should do it.